content/zh/docs/devops-user-guide/examples/a-maven-project.md

```diff
@@ -73,7 +73,7 @@ kubectl get cm -n kubesphere-devops-system ks-devops-agent -o yaml
 
 在本示例中，所有工作负载都部署在 `kubesphere-sample-dev` 项目中。您必须事先创建 `kubesphere-sample-dev` 项目。
 
-### 为 Maven 工程创建一个流水线
+### 为 Maven 项目创建一个流水线
 
 1. 在您的 DevOps 项目中，转到**流水线**页面并点击**创建**，创建一个名为 `maven` 的流水线。有关更多信息，请参见[使用图形编辑面板创建流水线](../../how-to-use/create-a-pipeline-using-graphical-editing-panel)。
```

Comment:
The link of `create-a-pipeline-using-graphical-editing-panel` should be `../../devops-user-guide/how-to-use/create-a-pipeline-using-graphical-editing-panel.md`

Reply:
Hi, thanks for your review. The link here is correct.